ETC.tsv

```diff
@@ -5158,7 +5158,7 @@ ETC_20150317_005157	Defeat Biteregina!
 ETC_20150317_005158	Small Hive
 ETC_20150317_005159	We are too late!
 ETC_20150317_005160	Wounded Villager
-ETC_20150317_005161	So you followed me here...
+ETC_20150317_005161	Were you looking for me?...
```

Comment:
This is completely different tone. I forgot what this was about, but *it* doesn't really question if you were looking for him/her. He basically confirmed that you were following him.